evap/evaluation/tools.py

```diff
@@ -116,3 +116,47 @@ def get_parameter_from_url_or_session(request, parameter, default=False):
 def translate(**kwargs):
     # get_language may return None if there is no session (e.g. during management commands)
     return property(lambda self: getattr(self, kwargs[get_language() or 'en']))
+
+
+def round_datetime(dt=None, round_to=60):
+    """Round a datetime object to closest time lapse (hours/minutes/seconds)
+    dt: datetime object, if is None: equals datetime.now()
+    round_to: time lapse in seconds, default: 1 minute
+    Microseconds will be discarded, rounding in days not supported"""
+
+    if dt is None:
+        dt = datetime.datetime.now()
+    seconds = (dt.replace(tzinfo=None) - dt.min).seconds
+    rounding = (seconds + round_to / 2) // round_to * round_to
+
+    return dt + datetime.timedelta(0, rounding - seconds, -dt.microsecond)
+
+
+def round_datetime_down(dt=None, round_to=60):
```

Comment:
python naming would be floor and ceil. I feel like having 3 function here is kind of overkill since basically, all three (floor, ceil, round) are just a different calculation in the original function when you are doing
`rounding = (seconds + round_to / 2) // round_to * round_to`. One would add `round_to`, one would add `round_to / 2`, one would not add anything.

Reply:
Also, the _down variant is unused currently, as well as the round variant itself, right? I vote to just make the one function we need, call it `ceil_datetime` and be done with it.